hub/core/storage/provider.py

```diff
@@ -94,6 +94,8 @@ def set_bytes(
                 value = value.rjust(end_byte, BYTE_PADDING)
             self[path] = value
 
+        self.maybe_flush()
```

Comment:
I don't think we need this here. It would already be flushed on line 90 or 95

Reply:
Yep, you're right. Good catch